File: C++/fizzbuzz.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    for (int i=1; i<=100; i++) {

        string output = "";

        if (i % 3 == 0) {
            output += "Fizz";
        };

        if (i % 5 == 0) {
            output += "Buzz";
        };

        if (output == "") {
            cout << i;
        } else {
            cout << output;
        };
        cout << "\n";
    }
    return 0;
}

File: Rust/fizzbuzz.rs
fn main() {
    for i in 1..101 {
        let mut output = String::from("");

        if i % 3 == 0 {
            output.push_str("Fizz");
        }

        if i % 5 == 0 {
            output.push_str("Buzz");
        }

        if output == "" {
            output = i.to_string();
        }
        
        println!("{}", output);
    }
}
